utils/src/com/goide/GoEnvironmentUtil.java

```diff
@@ -13,15 +12,22 @@
 
 public class GoEnvironmentUtil {
   public static final String GO_EXECUTABLE_NAME = "go";
-  
+  public static final String GAE_EXECUTABLE_NAME = "goapp";
+
   private GoEnvironmentUtil() {
   }
 
-  @NotNull
+  @Nullable
```

Comment:
Why do you need return null? Ok, file is not really exist, but by returning null you make client to check whether it null and handle this situation, whereas most of them don't have to know about it, they just create GeneralCommandLine and hope the everything will be fine.